### ISSUE TITLE
[Bug]: Broken queue after failed video.

### ISSUE THREAD

**Issue by pairofcrocs** [tubearchivist/tubearchivist#307]
### Latest and Greatest

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).

### Operating System

Unraid

## Describe the bug
After a download fails, trying to manually download another video causes ALL downloads to not work.

You can't select "Start Download" because that will download the already failed video.
Ignoring the failed video, still hangs on the next video in the queue.

## Steps To Reproduce
Download single video that was previously stuck (SER1RsBS_pY)

Press "Start Download"

Broke

## Expected behavior
It should just download the video without failing.

### Relevant log output

```shell
Pressing the "Start Download" button after the queue is broke, gives:

DcxpZA9tEyQ: downloading now
set task id: 4160ed19-52f7-4f09-905e-b2afdc190ca3
[2022-08-14 19:55:22,274: INFO/MainProcess] Task home.tasks.download_single[4160ed19-52f7-4f09-905e-b2afdc190ca3] received
[2022-08-14 19:55:22,276: WARNING/ForkPoolWorker-16] Added to queue with priority: DcxpZA9tEyQ
[2022-08-14 19:55:22,276: WARNING/ForkPoolWorker-16] Download queue already running.
[2022-08-14 19:55:22,277: INFO/ForkPoolWorker-16] Task home.tasks.download_single[4160ed19-52f7-4f09-905e-b2afdc190ca3] succeeded in 0.0018353695049881935s: None
```

### Anything else?

It seems that TA thinks that there is a video downloading when that's not the case.

**Comment by bbilly1** [comment]
So the interesting question would be, why the previous video failed to download? 

In any case, you can release any lingering locks by restarting the container, there is a check at startup. 

Then we'll need to find a better way on how to validate that, to see if the task is actually running or not...

**Comment by pairofcrocs** [comment]
Yeah the restart works, just knew that this wasn’t normal behavior :)

**Comment by p-rintz** [comment]
This is happening pretty often for me. At least once every two/three days.

I cant find any information in the logs about which video failed why.

A restart fixes the issue, but having to manually restart the container is getting tiresome after a while :smile: 

Is there anything I can provide that will help in debugging this?

**Comment by bbilly1** [comment]
This problem is now addressed in v0.3.5, please update.
